dspace-xmlui/dspace-xmlui-api/src/main/java/org/dspace/app/xmlui/aspect/authority/concept/ManageConceptMain.java

```diff
@@ -140,7 +143,14 @@ public void addBody(Body body) throws WingException, SQLException
         /* Get and setup our parameters */
         int page          = parameters.getParameterAsInteger("page",0);
         int highlightID   = parameters.getParameterAsInteger("highlightID",-1);
-        String query      = decodeFromURL(parameters.getParameter("query",null));
+        String query_1      = parameters.getParameter("query", null);
+        String query = "";
+        try{
+            query= URLDecoder.decode(query_1, Constants.DEFAULT_ENCODING);
+        }catch (UnsupportedEncodingException e)
+        {
+            log.error("decode error:"+e);
```

Comment:
It's better to write this with the exception passed as an argument, so the logs contain the stack trace and not just a simple message.

```log.error("decode error:", e);```